src/components/SideNav/TrackedContent.js

```diff
@@ -6,35 +6,23 @@ import { SideNavProgressContext } from "components/SideNav/Provider";
 
 const El = styled.div``;
 
-export const TrackedContent = ({ children }) => {
+export const TrackedContent = ({ children, identifier }) => {
```

Comment:
I called this `identifier` and not `id` in an attempt to clarify that it's not a DOM id. Not sure that's super helpful tho